pages/docs/concepts/asyncapi-document/_section.md

```diff
@@ -0,0 +1,4 @@
+---
+title: AsycnAPI-document
```

Comment:
Why did you put a dash there? Notice how weird it looks live, too 😂
<img width="207" alt="Screen Shot 2023-08-31 at 5 56 30 PM" src="https://github.com/asyncapi/website/assets/19964402/681ea691-e559-400d-90de-ba9303f9299d">